### DISCUSSION TITLE
Activity index is broken

### DISCUSSION THREAD

**Discussion by Paulito91** [plan-player-analytics/Plan#3257] [Q&A]
Hello. I have noticed that activity index in my server just broke. I'm using version 5.6 build 2556.
<img width="832" alt="image" src="https://github.com/plan-player-analytics/Plan/assets/6237123/91d862d0-55fd-41b2-8fd4-d0ab45a8a08c">
Any ideas how to fix this?
On the other hand playerbase statistics works as expected:
<img width="1015" alt="image" src="https://github.com/plan-player-analytics/Plan/assets/6237123/62b48c67-590a-48ba-8364-9c5a648e0d46">

**Comment by AuroraLS3** [comment]
Yes I noticed this as well has been fixed in a newer build https://github.com/plan-player-analytics/Plan/actions/runs/6465871762
